app/models/pawn.rb

```diff
@@ -1,10 +1,38 @@
 class Pawn < Piece
-  
   def self.get_image(color)
     if color == "white"
       return "chess_piece_pawn_white.png"
     elsif color == "black"
       return "chess_piece_pawn_black.png"
     end
   end
+
+  def valid_move?(x,y)
+    @object = self.game.pieces.where(position_x: x, position_y: y)[0]
```

Comment:
what is the purpose of @object? my guess is that it is a (possible) piece at the destination.. lets give it a meaningful name - maybe sth like `piece_on_target`